models/snapchat_ads__account_report.sql

```diff
@@ -0,0 +1,61 @@
+with ad_hourly as (
+
+    select *
+    from {{ var('ad_hourly_report') }}
+
+), account as (
+
+    select *
+    from {{ var('ad_account_history') }}
+    where is_most_recent_record = true
+
+), ads as (
+
+    select *
+    from {{ var('ad_history') }}
+    where is_most_recent_record = true
+
+), ad_squads as (
+
+    select *
+    from {{ var('ad_squad_history') }}
+    where is_most_recent_record = true
+
+), campaigns as (
+
+    select *
+    from {{ var('campaign_history') }}
+    where is_most_recent_record = true
+
+
+), aggregated as (
+
+    select
+        cast(ad_hourly.date_hour as date) as date_day,
+        account.ad_account_id,
+        account.ad_account_name,
+        account.currency,
+        sum(ad_hourly.swipes) as swipes,
+        sum(ad_hourly.impressions) as impressions,
+        round(sum(ad_hourly.spend)) as spend
```

Comment:
`roung()` without the precision defined will round to the nearest dollar. For example:
```txt
round(1.4445) = 1.0
```
To be safe, can we add a precision of 2 in this round statement. I don't want us to round down or up to the dollar. The 100th place seems comfortable though.

Reply:
Can you make this update to all other models in this transform package.

Reply:
Oh good good point, it definitely should be in the cents, not whole dollar!